ReportingServicesTools/Functions/CatalogItems/Rest/Get-RsRestItemDataSources.ps1

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsRestItemDataSources
```

Comment:
Our functions are typically singular (DataSource

Reply:
Removed this command for now. It was required for Set-RsRestItemDataSources, which I am pulling out from this PR.